rollup.build.js

```diff
@@ -53,11 +56,65 @@ async function createAllInclusiveBundles() {
   await fsWrite(`${UMD_DIR}/bundle.production.min.js`, production);
 }
 
+async function generateSizesFile() {
```

Comment:
There are two different versions of this method.
Thoughts on both?
I personally prefer to not use Promise.all very much and i like the size of the reducer.
They are almost identical in speed

![Screen Shot 2019-06-26 at 9 34 23 AM](https://user-images.githubusercontent.com/10228768/60198376-48c30480-97f6-11e9-86ae-92e4448974dc.png)